docs/fr/user/upgrade-2.0.x-2.1.1.rst

```diff
@@ -28,25 +26,45 @@ Téléchargez la dernière version de wallabag :
 
 .. code-block:: bash
 
-    wget http://wllbg.org/latest-v2-package && tar xvf latest-v2-package
```

Comment:
This page is for the 2.0.x -> 2.1.1 upgrade. So we need to hardcode this version.